src/index.js

```diff
@@ -16,6 +16,7 @@ if (!window.CMS_MANUAL_INIT) {
  * Add extension hooks to global scope.
  */
 if (typeof window !== 'undefined') {
+  window.React = React;
```

Comment:
We should check to ensure `window.React` isn't already defined, otherwise this could break setups that have their own instance on purpose.